save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/LnkContestProjectController.kt

```diff
@@ -114,7 +114,7 @@ class LnkContestProjectController(
         @PathVariable contestName: String,
         authentication: Authentication,
     ): Mono<List<String>> = Mono.fromCallable {
-        lnkUserProjectService.getAllProjectsByUserId((authentication.details as AuthenticationDetails).id).filter { it.public }
```

Comment:
As I remember, this request gives us the empty list, why `getCreatedProjectsByUserId` should help us, if the list of all projects are empty?

Reply:
The problem is that in prod database we have `DELETED` project with no organization while we somehow allow organization to be empty in `Project.kt` without setting organization type to be nullable